shared/src/hooks/useAssetPrice.ts

```diff
@@ -75,14 +75,14 @@ export const useFetchAssetsPrice = (
       };
     });
 
-    // Default USDC price
+    // Default USDC price for 3 years
     setHistoricalData((prev) => {
       return {
         ...prev,
         USDC: {
           loading: false,
           history: Object.fromEntries(
-            [...new Array(365)].map((_, index) => [
+            [...new Array(1095)].map((_, index) => [
```

Comment:
Nit: best to do 365*3 or something so it's more understandable at a glance

Reply:
will address in next commit